.changeset/breezy-papayas-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@strapi/icons': major
```

Comment:
```suggestion
'@strapi/icons': minor
```

Reply:
@simotae14 how come you initially set it as a `major` release?

Reply:
I didn't have any option to change it. I know it is not a major release

Reply:
I mean when I did the yarn command

Reply:
You 100% can, you just hit `enter` instead of selecting packages for the `major` release :) no worries tho